src/Nethermind/Nethermind.Trie.Test/Pruning/TreeStoreTests.cs

```diff
@@ -20,8 +20,8 @@ namespace Nethermind.Trie.Test.Pruning
     [TestFixture]
     public class TreeStoreTests
     {
-        private ILogManager _logManager = new OneLoggerLogManager(new NUnitLogger(LogLevel.Trace));
-
+        private readonly ILogManager _logManager = LimboLogs.Instance;
+            // new OneLoggerLogManager(new NUnitLogger(LogLevel.Trace));
```

Comment:
Did you miss this?

Reply:
No the tests are currently flooded with messages from those and while tests are fine they are not needed. They would only be useful when we have regression.